src/common/i18n.ts

```diff
@@ -11,6 +11,23 @@ import numbroCs from 'numbro/dist/languages/cs-CZ.min.js';
 import numbroDa from 'numbro/dist/languages/da-DK.min.js';
 import numbroLv from 'numbro/dist/languages/lv-LV.min.js';
 import numbroPl from 'numbro/dist/languages/pl-PL.min.js';
+import numbroEs from 'numbro/dist/languages/es-ES.min.js';
```

Comment:
Should this be `es-US`?

Reply:
Or are we treating our own "es-US" as Spanish but with US number formatting and currency?

Reply:
Yeah, numbro didn't have es-US out of the box, so I modified one from basic Spanish with some US conventions that seemed relevant sprinkled in. According to a quick Google search American Spanish has adopted some American conventions (see e.g. https://riuss.org/numerical-notation).

But to be honest I wasn't super confident about this and the whole commit in general, just tried to kind of copy what had been done earlier, so if you have more doubts please tell :smile:

Reply:
I would've done it the same! I don't quite understand the "Use locale directive in all GraphQL queries to make sure translated strings are always fetched" part though, I don't see any code related to that except removing it from the context parameter.

Or are we just now using the pre-existing `@locale` directive [here](https://github.com/kausaltech/kausal-paths-ui/blob/main/src/common/apollo.ts)?

Reply:
> I would've done it the same! I don't quite understand the "Use locale directive in all GraphQL queries to make sure translated strings are always fetched" part though, I don't see any code related to that except removing it from the context parameter.
> 
> Or are we just now using the pre-existing `@locale` directive [here](https://github.com/kausaltech/kausal-paths-ui/blob/main/src/common/apollo.ts)?

I'll answer in the other thread, this relates to that